service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+** *Step 3: Deploying an example application in OpenShift*
+** *Step 4: Creating a topic in your Kafka instance*
+* You should understand how the RHOAS CLI injects connection credentials as files into a client application Pod. To learn more, see xref:con-about-service-binding-using-rhoas-operator_{context}[].
 
 .Procedure
-. Execute quarkus application template that will be deployed to your namespace
+. If you are not already logged in to your OpenShift cluster, log in as a user that has privileges to deploy applications in the OpenShift project created earlier in this tutorial.
 +
-[source]
+[source, subs="+quotes"]
 ----
-oc apply -f https://raw.githubusercontent.com/redhat-developer/app-services-guides/main/code-examples/quarkus-kafka-quickstart/.kubernetes/kubernetes.yml
+$ oc login -u __<user>__ -p __<password>__ --server=__<host:port>__
 ----
-+
-. Fetch route to the application by executing
+. Ensure that the current OpenShift project is the one created earlier in this tutorial. For example:
 +
 [source]
 ----
-oc get route
+$ oc project rhoas-quarkus
 ----
-+
-. Open url from **HOST/PORT** section in your default browser
-. Verify that you see Quarkus landing page
-. If the Quarkus application fails to run, review the error log in the terminal and address any problems. Also review the steps in this quick start to ensure that the Quarkus application and Kafka topic are configured correctly.
-. Append `/prices.html` to url to see prices section.
-. You should see `N/A` as price as we still did not connected our application to our Kafka instance
-
-ifdef::qs[]
-.Verification
-* Did the Quarkus example application run without any errors?
-* Can you view `prices.html` page
-endif::[]
-
-[id="proc-binding-kafka_{context}"]
-== Connecting your service with running application
-
-Once application is running we can now connect our service using `rhoas cluster bind` command. This command will let us inject credentials to our application as files. Credentials will be by default injected as volume into our kuberentes deployment. Quarkus Kuberentes Client will detect them and automatically configure our application for us.
-
-Bind command will create following structure in your pod that many frameworks like quarkus can automatically read and autoconfigure your application to connect with the Kafka and other services.
 
+. Use the RHOAS CLI to bind your Kafka instance to your OpenShift project.
++
 [source]
 ----
-/bindings/yourkafka
-├── bootstrapServers
-├── password
-├── provider
-├── saslMechanism
-├── securityProtocol
-├── type
-└── user
+$ rhoas cluster bind
 ----
++
+You are prompted to specify the Kafka instance that you want to connect to OpenShift.
 
-.Procedure
-. Execute
+. Type the name of the Kafka instance that you want to connect to OpenShift. Press *Enter*.
+. Type `y` to confirm that you want to continue. Press *Enter*.
++
+When binding is complete, you should see output like the following:
```

Comment:
That would be better as verification step?

Reply:
@wtrocki A reminder that this content is **not** for a quick start. It is for a tutorial in a service binding guide. We don't include a **Verification** section, like we do in quick starts.

Reply:
I was asked and have written this content to appear in the cloud.redhat.com guides. 
Did something changed since then?

Reply:
Discussed in person with @wtrocki to clarify what we are delivering. We are delivering this content as part of a guide on c.rh.c, and not as a quick start.